games-rpg/eternal-lands/eternal-lands-1.9.5.3.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit desktop toolchain-funcs xdg
+
+DESCRIPTION="A 3D fantasy MMORPG written in C and SDL"
+HOMEPAGE="http://www.eternal-lands.com"
+SRC_URI="https://github.com/raduprv/Eternal-Lands/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="eternal_lands"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND="app-arch/unzip
+	dev-libs/libxml2:=
+	media-libs/cal3d:=[-16bit-indices]
+	media-libs/freealut
+	media-libs/libpng:0=
+	media-libs/libsdl:=[X,opengl,video]
+	media-libs/libvorbis:=
+	media-libs/openal:=
+	media-libs/sdl-image:=
+	media-libs/sdl-net:=
+	sys-libs/zlib:=[minizip]
+	virtual/glu
+	virtual/opengl
+	x11-libs/libX11:=
+	x11-libs/libXau:=
+	x11-libs/libXdmcp:=
+	x11-libs/libXext:="
```

Comment:
All of the `:=` slot operators (except possibly cal3d, don't know that one) are unnecessary and shouldn't be used because they slow Portage down significantly. zlib looks like it should have one but the subslot was added to this package by mistake.

Reply:
Yes, I thought about it, thanks. Wasn't sure, just copied from the previous version.
Looks like `:=` is not needed for cal3d either, it has no subslots.